### PR TITLE
Add coworking Jan/Feb

### PR DESCRIPTION
**Pull request by steffilazerte**
Coworking events for January and February

### CODE REVIEW

File: content/events/2024-01-09-coworking-2024-01.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+lubridate::with_tz(d, 'America/Winnipeg')
+```
+-->
+
+**Join us for 2 hours Tuesday January 09, 14:00 European Central (13:00 UTC) for 
+[Social Coworking + Office Hours](/blog/2023/06/21/coworking/)**
+
+This month we're focusing on the theme **Working with APIs** 
+with community host **Jon Harmon**, [R4DS](https://rfordatasci.com/) 
+Chief Community Manager, R package developer and API enthusiast!
```
Comment (jonthegeek) [inline content/events/2024-01-09-coworking-2024-01.md:39]:
```suggestion
Chief Community Manager, [R package developer](https://beekeeper.api2r.org/) and [API enthusiast](https://wapir.io/)!
```

**Review by jonthegeek — APPROVED** [review]
Update ORCID + add links for my API-related work

**Comment by steffilazerte** [comment]
Hmm, @jonthegeek 'https://wapir.io/' seems to be broken?

**Comment by jonthegeek** [comment]
Doh, it broke when I rebuilt... reminding myself what I need to do to avoid that...

**Pull request merged**